hera_corr_cm/hera_corr_cm.py

```diff
@@ -234,17 +237,21 @@ def _get_status_keys(self, stattype):
         """
         keystart = "status:{stat}:".format(stat=stattype)
         rv = {}
+        decode_responses = False if stattype == 'snap' else True
```

Comment:
and `snap` is the only key getting changed to byte arrays? Because now we use the same `status:snap:*` key for both antenna and snap stats?

Reply:
Yes - there is now one redis hash key to bind them all.